### DISCUSSION TITLE
Feature Request: Search Artists

### DISCUSSION THREAD

**Discussion by NyTimez** [navidrome/navidrome#3331] [Ideas]
Would you add a feature to be able to jump to artists with names that start with a specific letter?
# A B C D E F G H I J K L M N O P Q R S T U V W X Y Z

**Comment by deluan** [comment]
This will probably be available in the new UI. The Subsonic API already exposes this index, but AFAIK, all modern Subsonic clients ignores it.

**Reply by dweymouth** [reply]
It doesn't work well with non-Roman alphabet artist names, for one reason

**Reply by deluan** [reply]
Yes, it does not. But it is configurable, so potentially the user can group artists the way that makes sense for them.

The configuration option is `IndexGroups`, and the default is `A B C D E F G H I J K L M N O P Q R S T U V W X-Z(XYZ) [Unknown]([)`

I just realized this is not listed in the [configuration options page](https://www.navidrome.org/docs/usage/configuration-options/). I'll add it there, but in the meantime, here's how to configure it:

```
The specification is a space-separated list of index entries. Normally, each entry is just a single character,
but you may also specify multiple characters.  For instance, the entry "The" will link to all files and
folders starting with "The".

You may also create an entry using a group of index characters in parentheses. For instance, the entry
"A-E(ABCDE)" will display as "A-E" and link to all files and folders starting with either
A, B, C, D or E.  This may be useful for grouping less-frequently used characters (such and X, Y and Z), or
for grouping accented characters (such as A, \u00C0 and \u00C1)

Files and folders that are not covered by an index entry will be placed under the index entry "#".
```
